client-library/src/Attestation/LinuxTpm/tools/cmake/FindGTest.cmake

```diff
@@ -1,8 +1,8 @@
 # Try to find GTest
-find_path(GTEST_INCLUDE_DIR NAMES gtest PATHS ${SYSTEM_LIBRARY_PREFIX}/usr/local/include ${SYSTEM_LIBRARY_PREFIX}/usr/include NO_DEFAULT_PATH)
```

Comment:
we shouldn't remove NO_DEFAULT_PATH flag from here.

Reply:
gtest and gmock both installed in pre-requisites at default path.